### PR TITLE
Added examples how to use the new services

### PR DESCRIPTION
**Pull request by azrael783**
Added a template_cover and also an example for the `custom:button-card` for directly usage in lovelace. The link in "Supported devices" does point directly to the repo. Is there a document missing? I think we should fix this. Right now it is more confusing than helping ...

### CODE REVIEW

**Review by vlebourl — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -45,6 +45,43 @@ This integration is included in HACS. Search for the `Somfy TaHoma` integration
 
 ## Advanced
 
+### Examples
+#### Using a template_cover
```
Comment (iMicknl) [inline README.md:49]:
```suggestion
#### Use a template cover to control your cover with low speed
```

File: README.md [hunk 2]
```diff
@@ -45,6 +45,43 @@ This integration is included in HACS. Search for the `Somfy TaHoma` integration
...
+          target:
+            entity_id: cover.your_cover
+          data:
+            position: 100
+        close_cover:    
+          service: tahoma.set_cover_position_low_speed
+          target:
+            entity_id: cover.your_cover
+          data:
+            position: 20
```
Comment (iMicknl) [inline README.md:68]: Why is the position 20?

File: README.md [hunk 3]
```diff
@@ -45,6 +45,43 @@ This integration is included in HACS. Search for the `Somfy TaHoma` integration
 
 ## Advanced
 
+### Examples
+#### Using a template_cover
+If your cover supports the slow moving mode you can create template_covers to move the cover always in the slow mode. Add in `configuration.yaml` the following code, replacing the `cover.your_cover` with your entity name:
+```yml
+cover:
+  - platform: template
```
Comment (iMicknl) [inline README.md:53]: I have never used a cover template, but don't we need to add more properties? (https://www.home-assistant.io/integrations/cover.template/#covers).

For example: `stop_cover` and `set_cover_position`?

File: README.md [hunk 4]
```diff
@@ -45,6 +45,43 @@ This integration is included in HACS. Search for the `Somfy TaHoma` integration
...
+        close_cover:    
+          service: tahoma.set_cover_position_low_speed
+          target:
+            entity_id: cover.your_cover
+          data:
+            position: 20
+```
+Then add in your UI a button card calling the entity `cover.your_cover_slow`.
+
+#### Using the custom:button-card
```
Comment (iMicknl) [inline README.md:72]:
```suggestion
#### Map the 'my button' to a Home Assistant button
```

File: README.md [hunk 5]
```diff
@@ -45,6 +45,43 @@ This integration is included in HACS. Search for the `Somfy TaHoma` integration
...
+If you have HACS installed you can use the [button-card](https://github.com/custom-cards/button-card) to open and close your covers calling the new services. Create a new `button-card` in your UI and add the following code:
+```yml
+type: custom:button-card
+icon: mdi:window-open
+tap_action:
+  action: call-service
+  service: tahoma.set_cover_my_position
+  service_data:
+    entity_id: cover.gaestezimmer_1
```
Comment (iMicknl) [inline README.md:81]:
```suggestion
    entity_id: cover.gaestezimmer_1 # add your entity id here.
```

**Review by iMicknl — COMMENTED** [review]
Thanks again for your contribution! Some minor feedback.

**Pull request closed without merging**